lib/src/api/engine/remote/ws/native.rs

```diff
@@ -244,7 +245,8 @@ pub(crate) fn router(
 								}
 								let payload = Value::from(request);
 								trace!("Request {payload}");
-								Message::Binary(payload.into())
+								let payload = serialize(&payload).unwrap();
```

Comment:
I know it's very unlikely that we will run into a serialisation error here but now that we can handle it, I would like to do so. Any chance you can leave a `TODO` comment here so I can handle this later?